src/main.js

```diff
@@ -55,14 +56,115 @@ import { TrajectoryCalculator } from "./core/trajectoryCalculator.js";
     let playerTurn = true;
     let [playerOneMoveDist, playerTwoMoveDist] = [20, 20];
 
+    // creating the 'world' object to do physics calculations
+    let world = new World();
+ 
+    world.setGravity(Vec2(0, -9.8));
```

Comment:
can't we just set this as we create the world? Like:
```
let world = new World();
    .setGravity(...)   
```

Reply:
Nah yeah I can, I was just testing stuff